Cannot find name 'XRAnchor'

Hi,

Running into a few errors while building this.  Note that although there are errors, it does output a build successfully. 

Steps to reproduce:
- clone and cd to repo
- run `npm install`
- run `build:release` or `build:debug`
- Observe the errors below

```console
> playcanvas-editor-ts-template@1.1.0 build:release
> tsc --project tsconfig.release.json

node_modules/playcanvas/build/playcanvas.d.ts:33541:17 - error TS2552: Cannot find name 'XRAnchor'. Did you mean 'XrAnchor'?

33541     _index: Map<XRAnchor, XrAnchor>;
                      ~~~~~~~~

  node_modules/playcanvas/build/playcanvas.d.ts:33427:15
    33427 declare class XrAnchor extends EventHandler {
                        ~~~~~~~~
    'XrAnchor' is declared here.

node_modules/playcanvas/build/playcanvas.d.ts:33680:22 - error TS2304: Cannot find name 'XRReferenceSpace'.

33680     _referenceSpace: XRReferenceSpace | null;
                           ~~~~~~~~~~~~~~~~

types/global.d.ts:1:22 - error TS2792: Cannot find module 'playcanvas'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?

1 import * as _pc from 'playcanvas'
                       ~~~~~~~~~~~~

(⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂) ⠦ : timing config:load:flatten Completed in 6ms
Found 3 errors in 2 files.

Errors  Files
     2  node_modules/playcanvas/build/playcanvas.d.ts:33541
     1  types/global.d.ts:1

```


I have tried:
- Updating the `playcanvas` dependency to latest
- Using a different version of node
- Setting module resolution to node
    - This gets rid of the `Cannot find module 'playcanvas'.` error

# Workarounds:

## Option 1
### global.d.ts
```diff
import * as _pc from 'playcanvas'
declare global {
    const pc : typeof _pc;
+    interface XRAnchor {}
+    interface XRReferenceSpace {}
}
```


## Option 2
### tsconfig.x.json
```diff
{
  "compilerOptions": {
    "target": "ES5",
+    "moduleResolution": "node",
    "module": "amd",
    "lib": [
      "DOM",
      "ES2017"
    ],
    "outFile": "./build/main.build.js",
    "rootDir": "./src",
    "strict": true,
+    "skipLibCheck": true,
    "strictPropertyInitialization": false,
    "types": [
      "playcanvas"
    ],
    "esModuleInterop": true,
    "experimentalDecorators": true,
    "forceConsistentCasingInFileNames": true
  }
}


```

--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,5 @@
 
 node_modules
-package-lock.json
 build
 *.DS_Store
 pcconfig.json

--- a/package.json
+++ b/package.json
@@ -31,11 +31,12 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "playcanvas": "^1.59.0",
+    "@types/webxr": "^0.5.5",
+    "playcanvas": "^1.66.1",
     "playcanvas-sync": "git+https://github.com/playcanvas/playcanvas-sync.git",
-    "typescript": "^4.9.4"
+    "typescript": "^5.2.2"
   },
   "dependencies": {
-    "tsc-watch": "^5.0.3"
+    "tsc-watch": "^6.0.4"
   }
 }

--- a/tsconfig.debug.json
+++ b/tsconfig.debug.json
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "ES5",
     "module": "amd",
+    "moduleResolution": "node",
     "lib": [
       "DOM",
       "ES2017"
@@ -10,10 +11,6 @@
     "rootDir": "./src",               
     "strict": true, 
     "strictPropertyInitialization": false,
-    "types": [
-      "playcanvas"
-    ],
-    "typeRoots": ["./types/"],
     "esModuleInterop": true, 
     "inlineSourceMap": true, 
     "inlineSources": true,   

--- a/tsconfig.release.json
+++ b/tsconfig.release.json
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "ES5",
     "module": "amd",
+    "moduleResolution": "node",
     "lib": [
       "DOM",
       "ES2017"
@@ -10,9 +11,6 @@
     "rootDir": "./src",                
     "strict": true, 
     "strictPropertyInitialization": false, 
-    "types": [
-      "playcanvas"
-    ],
     "esModuleInterop": true,
     "experimentalDecorators": true, 
     "forceConsistentCasingInFileNames": true

